pickem/pickem_homepage/templates/pickem/base.html

```
@@ -152,137 +199,164 @@
+                        {% if active_banner.description %}<span class="text-white/85"> &mdash; {{ active_banner.description }}</span>{% endif %}
+                    </div>
+                    {% if active_banner.show_close_button %}
+                    <button type="button" class="banner-close -mr-1 flex-shrink-0 rounded-lg p-1.5 text-white/80 hover:bg-white/20 hover:text-white transition-colors" aria-label="Dismiss banner">
+                        <i class="fas fa-times"></i>
+                    </button>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+        {% elif banner_message or banner_type %}
+        <div id="site-banner" class="site-banner text-white shadow-sm {% if banner_type == 'success' %}bg-green-600{% elif banner_type == 'warning' %}bg-amber-600{% elif banner_type == 'danger' %}bg-red-600{% else %}bg-blue-600{% endif %}">
```

HTMLHint (1.9.2)

[error] The id value [ site-banner ] must be unique.

(id-unique)